mobile/Ihc/services/LocalDataService.js

```diff
@@ -399,6 +511,70 @@ export function markMedicationsUploaded() {
   });
 }
 
+export function markLabRequestsUploaded() {
+  const labRequests = Object.values(realm.objects('LabRequest').filtered('needToUpload = true'));
+  realm.write(() => {
+    labRequests.forEach(labRequest => {
+      labRequest.needToUpload = false;
+    });
+  });
+}
+
+export function handleDownloadedLabRequests(labRequests) {
```

Comment:
I highly recommend tests for these more complex methods like this one: https://github.com/TritonSE/ihc/blob/master/mobile/Ihc/__tests__/LocalDataService.js#L130
I won't force you, but especially once you pass this on, I'm quite certain someone may break it at some point